quantum/rgb_matrix.c

```diff
@@ -38,6 +38,7 @@
 #include "rgb_matrix_animations/rainbow_pinwheels_anim.h"
 #include "rgb_matrix_animations/rainbow_moving_chevron_anim.h"
 #include "rgb_matrix_animations/jellybean_raindrops_anim.h"
+#include "rgb_matrix_animations/typing_headmap_anim.h"
```

Comment:
Typo in filename 😉

Reply:
good catch!

Reply:
Well, the file name is correct. As that's the files actual name.

Reply:
Can't add reviews on filenames, I had to make do 😄

Reply:
will fix it in the cleanup pr I am working on now